tests/unittests/media/server/ipc/webAudioPlayerModuleService/WebAudioPlayerModuleServiceTestsFixture.cpp

```diff
@@ -18,6 +18,7 @@
  */
 
 #include "WebAudioPlayerModuleServiceTestsFixture.h"
+#include "MediaPipelineProtoUtils.h"
```

Comment:
Should be WebAudio & you need to delete the convertWebAudioPlayerState that's also defined in here.